src/System.IO.Compression/src/System/IO/Compression/DeflateStream.cs

```diff
@@ -18,124 +18,106 @@ public partial class DeflateStream : Stream
         private Inflater _inflater;
         private Deflater _deflater;
         private byte[] _buffer;
-
         private int _asyncOperations;
-
-        private IFileFormatWriter _formatWriter;
-        private bool _wroteHeader;
         private bool _wroteBytes;
 
-        public DeflateStream(Stream stream, CompressionMode mode)
-            : this(stream, mode, false)
+        #region Public Constructors
+
+        public DeflateStream(Stream stream, CompressionMode mode): this(stream, mode, false, ZLibNative.Deflate_DefaultWindowBits)
```

Comment:
Nit: personal style preference, but I prefer to delegate to the next overload that has the one argument I want to add rather than having to specify defaults in every overload, e.g. this would be:
```C#
: this(stream, mode, false)
```
rather than:
```C#
: this(stream, mode, false, ZLibNative.Defalte_DefaultWindowBits)
```
There shouldn't be any perf difference, and it avoids needing to specify these values in multiple places.

Your call.